src/lineage/entrypoint.py

```diff
@@ -333,42 +344,47 @@ def main() -> int:
-                    "remote_branch": remote_branch,
-                    "remote_url": remote_url,
-                    "repo_name": repo.name,
-                    "ssh_url": repo.ssh_url,
-                }
-                if conflict_file_list:
-                    title = f"⚠️ CONFLICT! Lineage pull request for: {lineage_id}"
-                    body = pystache.render(conflict_template, template_data)
-                    create_pull_request(
-                        repo, pr_branch_name, local_branch, title, body, draft=True
-                    )
+            # If we can successfully push, continue with generating a PR
+            if push(repo, pr_branch_name, "git", access_token):
+                # Display instructions for ignoring incoming Lineage config changes
+                display_lineage_config_skip: bool = CONFIG_FILENAME in conflict_file_list
+                if display_lineage_config_skip:
+                    # This will no longer be a conflict, we tell user how to ignore.
+                    conflict_file_list.remove(CONFIG_FILENAME)
+                if branch_is_new:
+                    logging.info("Creating a new pull request since branch is new.")
+                    template_data = {
+                        "conflict_file_list": conflict_file_list,
+                        "display_lineage_config_skip": display_lineage_config_skip,
+                        "lineage_config_filename": CONFIG_FILENAME,
+                        "lineage_id": lineage_id,
+                        "local_branch": local_branch,
+                        "metadata": PR_METADATA,
+                        "pr_branch_name": pr_branch_name,
+                        "remote_branch": remote_branch,
+                        "remote_url": remote_url,
+                        "repo_name": repo.name,
+                        "ssh_url": repo.ssh_url,
+                    }
+                    if conflict_file_list:
+                        title = f"⚠️ CONFLICT! Lineage pull request for: {lineage_id}"
+                        body = pystache.render(conflict_template, template_data)
+                        create_pull_request(
+                            repo, pr_branch_name, local_branch, title, body, draft=True
+                        )
+                    else:
+                        title = f"Lineage pull request for: {lineage_id}"
+                        body = pystache.render(clean_template, template_data)
+                        create_pull_request(
+                            repo, pr_branch_name, local_branch, title, body, draft=False
+                        )
                 else:
-                    title = f"Lineage pull request for: {lineage_id}"
-                    body = pystache.render(clean_template, template_data)
-                    create_pull_request(
-                        repo, pr_branch_name, local_branch, title, body, draft=False
+                    logging.info(
```

Comment:
Should we make this a warning?

Reply:
Let's make it a warning, but make sure the warning string starts with "ERROR!'